FirebaseAuth/Sources/Auth/FIRAuth.m

```diff
@@ -462,7 +464,8 @@ - (instancetype)initWithApp:(FIRApp *)app {
   self = [self initWithAPIKey:app.options.APIKey
                       appName:app.name
                         appID:app.options.googleAppID
-              heartbeatLogger:app.heartbeatLogger];
+              heartbeatLogger:app.heartbeatLogger
+                     appcheck:FIR_COMPONENT(FIRAppCheckInterop, app.container)];
```

Comment:
nit: Here and in the rest of the PR, can you please change `appcheck` to `appCheck`?

Reply:
yes, Have changed